tests/test_context.py

```diff
@@ -215,6 +215,34 @@ def tearDown(self):
         if os.path.exists(self.tempdir):
             shutil.rmtree(self.tempdir)
 
+    def test_down_chunking(self):
```

Comment:
Not exactly sure how you would like to organize tests, but the comment on this test class says it's about testing per-run defaults of the context. I guess that's outdated, but it looks like it's least about texting context-related stuff, not particular plugin types.

Plugin things seem to be tested in test_core.py or in separate test files (test_overlap_plugin.py, test_multi_output.py), where they make dummy contexts when needed.